src/main/java/roomescape/payment/domain/PaymentRestClient.java

```diff
@@ -48,9 +49,10 @@ public void approvePayment(PaymentCreateRequest paymentCreateRequest) {
                     .onStatus(HttpStatusCode::isError, ((request, response) ->
                             handleErrorMessage(response)
                     ))
-                    .toBodilessEntity();
+                    .body(RestClientPaymentApproveResponse.class);
         } catch (RestClientException e) {
```

Comment:
다시 읽다보니 좀 헷갈렸어요. (여기서 API 400 / 500 오류 핸들링이 되는걸로 인식 됨)
결국 400 / 500 에러는 onStatus 처리에서 해소 될 것이니, 이 부분은 최종 안전선으로 Exception e 처리를 해주는 정도로 마무리 하는것도 괜찮을거 같습니다. 

```suggestion
        } catch (Exception e) {
```